railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -85,6 +85,8 @@ RUN apt-get update -qq && \
     apt-get install --no-install-recommends -y <%= dockerfile_deploy_packages.join(" ") %> && \
     rm -rf /var/lib/apt/lists /var/cache/apt/archives
 
+ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
```

Comment:
This won't work if you build on ARM64 (typically M1 mac) or deploy to ARM servers (e.g. Graviton).

Reply:
@byroot there doesn't seem to be a way to interpolate commands in the ENV directive - we might have to move this to the `docker-entrypoint`.

Reply:
Yeah, that's where I meant to put the `jemalloc` command. But I misremembered anyway, the package doesn't include it.

I'll add the entrypoint, it shouldn't be too hard.